patches/vfile-message+2.0.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/vfile-message/types/index.d.ts b/node_modules/vfile-message/types/index.d.ts
+index d217843..31c853a 100644
+--- a/node_modules/vfile-message/types/index.d.ts
++++ b/node_modules/vfile-message/types/index.d.ts
+@@ -6,7 +6,7 @@ declare namespace vfileMessage {
+   /**
+    * Create a virtual message.
+    */
+-  interface VFileMessage extends Error {
++  interface VFileMessage extends Omit<Error, 'location'> {
```

Comment:
https://github.com/vfile/vfile-message/issues/6 😕

Reply:
alternate is to use `skipLibCheck` which disables typescript checks on external libs 😕

Reply:
I would prefer the new `2.x` release of `vfile-message`. It looks like the owner of `unified` is holding off for their next version bump: https://github.com/unifiedjs/unified/issues/121

Reply:
Yeah, I noticed that as well 😑

Reply:
So only patch-package or skiplibcheck are working here.

Reply:
@JamieMagee Some progress at https://github.com/vfile/vfile-message/issues/6 maybe we can fix types external before or after this soon.

Reply:
fixed